cluster/gce/gci/configure-helper.sh

```diff
@@ -3315,16 +3315,30 @@ oom_score = -999
   default_runtime_name = "runc"
 [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc]
   runtime_type = "io.containerd.runc.v2"
-[plugins."io.containerd.grpc.v1.cri".registry.mirrors."docker.io"]
-  endpoint = ["https://mirror.gcr.io","https://registry-1.docker.io"]
-# Enable registry.k8s.io as the primary mirror for k8s.gcr.io
-# See: https://github.com/kubernetes/k8s.io/issues/3411
-[plugins."io.containerd.grpc.v1.cri".registry.mirrors."k8s.gcr.io"]
-  endpoint = ["https://registry.k8s.io", "https://k8s.gcr.io",]
 [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc.options]
   SystemdCgroup = ${systemdCgroup}
+# enable hosts config
+[plugins."io.containerd.grpc.v1.cri".registry]
+  config_path = "/etc/containerd/certs.d"
 EOF
 
+  # DO NOT MERGE -- Testing
+  KUBERNETES_REGISTRY_PULL_THROUGH_HOST='https://us-west2-docker.pkg.dev/v2/k8s-artifacts-prod/images/'
```

Comment:
should probably instead test it with the planned private endpoint and tweak IAM to allow the other boskos projects ... but we can start with this test commit and public read.

NOTE: do not do this yourself, at some point we're going to lock down access to these backends :-)